From b44a2ce2474ad405ae8427a665bef42b527760c3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Joosep=20J=C3=A4=C3=A4ger?= <joosep.jaager@gmail.com>
Date: Sun, 24 Apr 2022 17:34:16 +0300
Subject: [PATCH] Fixed

---
 SS14.Launcher/Models/Connector.cs | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/SS14.Launcher/Models/Connector.cs b/SS14.Launcher/Models/Connector.cs
index a4bd3c2..9a0b340 100644
--- a/SS14.Launcher/Models/Connector.cs
+++ b/SS14.Launcher/Models/Connector.cs
@@ -411,17 +411,9 @@ public class Connector : ReactiveObject
         const bool release = false;
 #endif
 
-        if (release)
-        {
-            basePath = Path.Combine(LauncherPaths.DirLauncherInstall, "loader");
-        }
-        else
-        {
-            basePath = Path.GetFullPath(Path.Combine(
-                LauncherPaths.DirLauncherInstall,
-                "..", "..", "..", "..",
-                "SS14.Loader", "bin", "Debug", "net6.0"));
-        }
+        basePath = Path.GetFullPath(Path.Combine(
+            LauncherPaths.DirLauncherInstall,
+            "..", "..", "bin"));
 
         if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
         {
-- 
2.35.1

